src/Maestro/DependencyUpdateErrorProcessor/Program.cs

```diff
@@ -9,11 +9,14 @@
 using Microsoft.EntityFrameworkCore;
 using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using Microsoft.TeamFoundation.TestManagement.WebApi;
+using Microsoft.VisualStudio.Services.CircuitBreaker;
 using Octokit;
 
 namespace DependencyUpdateErrorProcessor
 {
-    public static class Program
+    public class Program
```

Comment:
Does this need to change?

Reply:
For this statement
provider.GetRequiredService<ILogger<Program>>(); static types cannot be used as type arguments. So I changed the class to non static.

Reply:
Uhmm. I'm not sure if this change really affects that error message. The _type argument_ here is ILogger and not Program. Can you please double check?

Reply:
ah I think I did a typo here 

var logger = provider.GetRequiredService<ILogger<Program>>();

It takes program as a argument.

Reply:
static types can't ever be passed as generic type arguments. For example, you can't have a List<Program>, because it doesn't make much sense in general.

Logger is weird in that it never really _uses_ the type argument you give it for anything really, it's just using it as a way to cheat a string into the DI system.

You could go GetRequiredService<ILoggerProvider>().CreateLogger(nameof(Program)).  Then you wouldn't have to de-static the class. (And that's really all the logging infrastructure is doing under the covers anyway)

Reply:
I was not really sure about making it de-static. This def is much cleaner. I agree that making the program class non static is weird. DI is magic :(